core/kazoo_data/src/kz_datamgr.erl

```diff
@@ -345,7 +345,26 @@ db_exists_all(DbName) ->
 -spec db_info() -> {'ok', kz_term:ne_binaries()} |
                    data_error().
 db_info() ->
-    kzs_db:db_info(kzs_plan:plan()).
+    Connections = get_connections(),
+    Databases = build_db_list(Connections),
+    {'ok', Databases}.
+
+-spec get_connections() -> map().
+get_connections() ->
+    [#{tag => Tag, server => kz_dataconnections:get_server(Tag)}
+     || {_, _, _, _, _, _, _, Tag}
+            <- ets:tab2list('kz_dataconnections')].
```

Comment:
Drop the trailing `]` down

Reply:
I feel like this should be a call to kz_dataconnections instead of a read from the ETS table.

Reply:
Ignore this for now, this was a poc to discuss and not the acutal implementation.

 I've renamed with WIP for now and will update when its ready for review.

Reply:
Meaning `kz_datamgr` doesn't need to know about the implementation details of how connections are tracked (via ETS or in a gen_server or whatever). `kz_datamgr` should ask kz_dataconnections for a list of active ones and go from there.

Also, have you tested with a storage plan added? Would be a good test to make sure it isn't included in the listing or crash the call somehow.

Reply:
OK thanks